docs/advanced/plugins/index.md

```diff
@@ -22,6 +22,8 @@ redirect_from:
   </p>
 </blockquote>
 
+You can also find official plugins at https://github.com/babel-plugins and other community plugins by searching [npm](https://www.npmjs.com/search?q=babel-plugin-).
```

Comment:
Ah ok cool it does commit to the same branch! I guess the only issue is you can't squash into a single commit without doing that locally and force pushing